resolwe/flow/models.py

```diff
@@ -105,6 +105,13 @@ class Meta(BaseModel.Meta):
 
     data = models.ManyToManyField('Data')
 
+    #: collection descriptor schema
+    descriptor_schema = models.ForeignKey('DescriptorSchema', blank=True, null=True, on_delete=models.PROTECT)
```

Comment:
At the moment we can not delete `Descriptor` if it is referenced in any Data object, so we have consistent descriptors and their schemes. If we change `models.PROTECT`, that means that we will have to handle descriptors without schemes. Do we really want this?